kani-driver/src/args.rs

```diff
@@ -166,6 +170,7 @@ impl KaniArgs {
 
     pub fn assertion_reach_checks(&self) -> bool {
         // Turn them off when visualizing an error trace.
+        // TODO: disable this for exec_trace flag?
```

Comment:
You probably want to keep them since you are still printing the verification report. You should however ignore the traces that get printed for them.